src/Game.js

```diff
@@ -428,22 +211,15 @@ Game.prototype = {
 	},
 
 	updateloop: function(timestamp) {
-
 		// check all timers
 		for (var t=0; t < this.timers.length; t++) {
 			if (this.timers[t].enabled) {
 				this.timers[t].update(timestamp);
 			}
 		}
-
-		// any shapes updates
-		if (this._refresh) {
-			this.shapesRefresh();
-			this._refresh = false;
-		}
 	},
 
-	gameReset: function(gametype) {
+	gameReset: function(gametype = this.gametype) {
```

Comment:
Some games use `this.gametype` as a value, thought it might make sense to make it a default.